share/normalize/links.py

```diff
@@ -840,6 +848,30 @@ def _parse(self, obj):
         }
 
 
+class ARKLink(AbstractIRILink):
```

Comment:
We don't get enough information to. They're usually just `ark:/1234/blah/blah`, and there's no single host that can resolve any ARK, unlike DOIs. They're useful as identifiers, but not locators.